### PR TITLE
Axios CVE

### PR DESCRIPTION
**Pull request by mtho11**
** Describe the change **

Update snyk for Axios CVE.

** Issue reference **
https://issues.jboss.org/browse/KIALI-2823

** Backwards compatible? **
yes

[ ] Is your pull-request introducing changes in behaviour?
No new behaviour

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,7 +101,8 @@
     "ssri": "6.0.1",
     "typesafe-actions": "3.2.1",
     "typestyle": "2.0.1",
-    "visibilityjs": "2.0.2"
+    "visibilityjs": "2.0.2",
+    "snyk": "^1.161.1"
```
Comment (josejulio) [inline package.json]: Isn't this a dev-dependency?

Comment (mtho11) [reply package.json]: So `snyk` removed it from devDependency and put it in dependency.

Comment (mtho11) [reply package.json]: But I agree, I don't see why it is needed as a dependency and it should be a dev-dependency instead.

Comment (mtho11) [reply package.json]: Changes pushed

**Pull request merged**